behat.yml.dist

```diff
@@ -3,41 +3,9 @@ imports:
 
 default:
     extensions:
-        FriendsOfBehat\ContextServiceExtension:
-            imports:
-                - "vendor/sylius/sylius/src/Sylius/Behat/Resources/config/services.xml"
-
+        Behat\MinkExtension:
```

Comment:
Could we copy Sylius/Sylius's `behat.yml.dist` contents here? So that we won't import `vendor/sylius/sylius/behat.yml.dist` here but only the suites definitions.